dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -130,3 +130,17 @@ OPEN_API_DOC_PATH_INVALID_CWWKO1676E.useraction=Ensure the OpenAPI Path contains
 OPEN_API_PATH_SEGMENT_INVALID_CWWKO1677E=CWWKO1677E: Path segment "/." or "/.." is invalid for OpenAPI endpoints.
 OPEN_API_PATH_SEGMENT_INVALID_CWWKO1677E.explanation=OpenAPI path segment "/." or "/.." is invalid.
 OPEN_API_PATH_SEGMENT_INVALID_CWWKO1677E.useraction=Remove any path "/." or "/.." segment from the path.
+
+# {1} - the server.xml element name, either includeApplication or excludeApplication
+OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W=CWWKO1678W: The configured {0} application name in the {1} element is not valid because it contains the "/" character.
+OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W.explanation=explanation: An application cannot name cannot include a "/" as "/" is used to separate the application from its component modules.
+OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W.useraction=useraction: Remove the "/" from the includeApplication or excludeApplication element in your server.xml, or convert that element to includeModule or excludeModule to reference a single module.
+
+# {1} - the server.xml element name, either includeModule or excludeModule
+OPEN_API_SLASH_IN_MODULE_CWWKO1679W=CWWKO1679W: The configured {0} module name in the {1} element is not valid it is not in the form "<application-name>/<module-name>".
+OPEN_API_SLASH_IN_MODULE_CWWKO1679W.explanation=explanation: An module must be in the format <application-name>/<module-name>, therefore a module name without a "/" is invalid.
+OPEN_API_SLASH_IN_MODULE_CWWKO1679W.useraction=useraction: Correct the includeModule or excludeModule element in your server.xml, or convert that element to includeApplication or excludeApplication to reference a whole application.
```

Comment:
```suggestion
# {1} - the server.xml element name, either includeApplication or excludeApplication
OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W=CWWKO1678W: The configured {0} application name in the {1} element is not valid because it contains the "/" character.
OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W.explanation=An application cannot name cannot include a "/" as "/" is used to separate the application from its component modules.
OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W.useraction=Remove the "/" from the includeApplication or excludeApplication element in your server.xml, or convert that element to includeModule or excludeModule to reference a single module.

# {1} - the server.xml element name, either includeModule or excludeModule
OPEN_API_SLASH_IN_MODULE_CWWKO1679W=CWWKO1679W: The configured {0} module name in the {1} element is not valid it is not in the form "<application-name>/<module-name>".
OPEN_API_SLASH_IN_MODULE_CWWKO1679W.explanation=An module must be in the format <application-name>/<module-name>, therefore a module name without a "/" is invalid.
OPEN_API_SLASH_IN_MODULE_CWWKO1679W.useraction=Correct the includeModule or excludeModule element in your server.xml, or convert that element to includeApplication or excludeApplication to reference a whole application.
```